usaspending_api/api_docs/api_documentation/financial_spending/object_class.md

```diff
@@ -0,0 +1,68 @@
+
+## Financial Spending Major Object Class
+**Route**: `/api/v2/financial_spending/object_class?fiscal_year=[year]&funding_agency_id=[id]&major_object_class_code=[id]`
+
+Retrieve Award Spending Amounts for all Recipients with Respective Top Tier and Sub Tier Agencies
```

Comment:
Not relevant to endpoint functionality. Should be: `Retrieve award obligation amounts broken down by object class`

Reply:
added